spec/api_spec.rb

```diff
@@ -526,6 +524,24 @@
       expect(bp).to be_truthy
     end
   end
+  describe 'folders' do
+    it 'should include max_results and next_cursor for root_folder call' do
```

Comment:
Move these tests closer to "should support listing folders" or even add them there